client/src/components/DevMode/context/useDevModeContext.tsx

```diff
@@ -0,0 +1,66 @@
+import { useContext, createContext, useState, ReactElement, FunctionComponent, useCallback } from 'react';
+import { AccountType } from '../../../types/AccountType';
+import login from '../../../helpers/APICalls/login';
+import register from '../../../helpers/APICalls/register';
+import { useAuth } from '../../../context/useAuthContext';
+
+export type AccountTypeKeys = keyof typeof AccountType;
+
+interface IDevModeStaticContext {
```

Comment:
'I' prefix is not necessary. It was a controversial topic, and you can read a lengthy discussion [here](https://github.com/microsoft/TypeScript-Handbook/issues/121)

Reply:
Understood, I will make the changes as I understand it is not necessary functionality. I was just following the overall structure of the project and the other contexts that were created. As the useAuth context makes use the same naming convention.